### ISSUE TITLE
Update info page

### ISSUE THREAD

**Issue by AlbertoCasasOrtiz** [opensim-org/opensim-viewer#108]
Related to #63, we should choose a license (addbiomechanics and sit2stand are using CC-BY 4.0 and CC BY-NC 4.0 if I'm not wrong) and update the info page.

Some of the models we use come from opensim, and I've noticed [the license](https://simtk-confluence.stanford.edu:8443/plugins/servlet/mobile?contentId=3376101#content/view/3376101) they use is a outdated ([CC By 3.0 unported](http://creativecommons.org/licenses/by/3.0/)). Should we update this as well? If so, we could create an issue in opensim-models.

**Comment by aymanhab** [comment]
Yes @AlbertoCasasOrtiz please open an issue and we'll discuss with @jenhicks as she's more on top of the various licenses/versions we use.
